src/main/scala/dotvisualizer/FirrtlDiagrammer.scala

```diff
@@ -7,6 +7,7 @@ import java.io.{File, PrintWriter}
 import chisel3.experimental.ChiselAnnotation
 import dotvisualizer.transforms.{MakeDiagramGroup, ModuleLevelDiagrammer}
 import firrtl._
+import firrtl.stage.TargetDirAnnotation
```

Comment:
Why is this now required? I don't see any new references to it.

Reply:
The TargetDirAnnotation lives in two places, one is deprecated. This overrides the `import firrtl.annotations._` that brings in the deprecated one.